addons/digest/views/digest_views.xml

```diff
@@ -56,8 +56,7 @@
                                 <group name="kpi_sales"/>
                                 <group name="custom" string="Custom" groups="base.group_system">
                                     <p>Want to add your own KPIs?<br />
-                                    <!-- TODO: change to actual documentation link!!! -->
-                                    <a href="https://www.odoo.com/documentation/" target="_blank"><i class="fa fa-arrow-right"></i> Check our Documentation</a></p>
+                                    <a href="https://www.odoo.com/documentation/latest/applications/general/digest_emails.html#custom-digest-emails" target="_blank"><i class="fa fa-arrow-right"></i> Check our Documentation</a></p>
```

Comment:
Question for reviewer: should this be /latest or /saas-15.1? I see that in the saas-15.1 branch all links have /saas-15.1... is this done automatically?

Reply:
It should be `/saas-15.1` and you should update it to `saas-15.2` and `master` as forward-port PRs are created. We're planning to have a widget handle this automatically but, for now, you need to perform this manually^^

`/latest` targets the latest major release of Odoo. (here 15.0)